### ISSUE TITLE
Translation error in "Showing" text box

### ISSUE THREAD

**Issue by lacroixca** [Automattic/WP-Job-Manager#647]
When filtering with -for example- job types in French, the showing text is wrong because of the way it is coded.
#### What Happens

It shows: 

> Afficher tous les **postes** Temps plein & Temps partiel **postes**

(poste = job, displayed twice)
#### What I Expected To Happen

It should show: 

> Afficher tous les **postes** Temps plein & Temps partiel
#### Code causing the problem

includes/class-wp-job-manager-ajax.php:178
`$result['showing'][] = $post_type_label;

if ( $search_location ) {
    $result['showing'][] = sprintf( __( 'located in &ldquo;%s&rdquo;', 'wp-job-manager' ), $search_location );
}

if ( 1 === sizeof( $result['showing'] ) ) {
    $result['showing_all'] = true;
}

$result['showing'] = apply_filters( 'job_manager_get_listings_custom_filter_text', sprintf( __( 'Showing all %s', 'wp-job-manager' ), implode( ' ', $result['showing'] ) ) );`
#### Solution

Do not add the Job label at the end of the text and change the sprintf() text:
`//$result['showing'][] = $post_type_label; //REMOVE THIS LINE

if ( $search_location ) {
    $result['showing'][] = sprintf( __( 'located in &ldquo;%s&rdquo;', 'wp-job-manager' ), $search_location );
}

if ( 0 === sizeof( $result['showing'] ) ) { //CHANGE HERE
    $result['showing_all'] = true;
}

$result['showing'] = apply_filters( 'job_manager_get_listings_custom_filter_text', sprintf( __( 'Showing all %sjobs', 'wp-job-manager' ), implode( ' ', $result['showing'] ) ) ); //ADD THE LABEL IN A TRANSLATABLE STRING`

**Comment by donnchawp** [comment]
There's a bug in the translation itself, but you're right that the post_type_label should be translated.

The translated string is "Afficher tous les postes %s" where the English string is "Showing all %s".

**Comment by donnchawp** [comment]
I fixed it by rewriting this string in #776. We don't need to show the search terms as they're already displayed on the page. It's more useful to show a summary of the search so it shows the post type (Jobs) and the number of records found.
